src/components/DetailView/DetailView.svelte

```diff
@@ -59,11 +65,42 @@
       }).then((rows) => addMissing(rows, sensor))
     : [];
 
+  // The currently selected date range, initially defaults to the smallMultipleTimeSpan.
+  // Modified by onDateRangeChange, and fetched via (non-reactive) getDateRange,
+  // to avoid updating the chart immediately, so it will be used as the initial value the
+  // next time createSpec is called.
+  // $: dateRange = dateRange || $smallMultipleTimeSpan;
+
+  function getDateRange() {
+    if ($detailViewTimeSpan.length === 0) {
+      detailViewTimeSpan.set($smallMultipleTimeSpan);
+    }
+    return $detailViewTimeSpan;
+  }
+
+  function onDateRangeChange(event) {
+    // All signal events are sent to all on:signal handlers (??), so check if this is the right handler.
+    if (event.detail.name !== 'dateRange') {
+      return;
+    }
+    const dr = event.detail.value && event.detail.value.date_value;
+    if (dr && Array.isArray(dr) && dr.length > 0) {
+      console.info('onDateRangeChange', dr);
```

Comment:
can you remove the log statement, please

Reply:
Yes, well, my thought was that it helps to have the log messages while trying to debug why it fails the first time.  I'll remove them before this is ready to merge.

Reply:
then please convert this PR to a draft such that I know it is not ready to be reviewed/merged. To my understanding if I'm assigned as a reviewer and the PR is not a draft, I assume I should review it because it is in its final state

Reply:
I needed your feedback (which was useful, by the way), and assumed you would want to see the console messages as well.  

But I think the 'draft' status is not quite right, unless I mean to say "you can ignore this PR until further notice", which is not what I wanted to convey.   We would need another status that indicates "Not ready to submit, but your feedback is needed".   And regardless, even if it is not in 'draft' mode, the PR would not be ready to submit until approved, so it means the same thing.